incubator/hnc/internal/reconcilers/object.go

```diff
@@ -538,10 +538,37 @@ func (r *ObjectReconciler) writeObject(ctx context.Context, log logr.Logger, ins
 	var err error = nil
 	stats.WriteObject(inst.GroupVersionKind())
 	if exist {
-		log.Info("Updating")
+		log.Info("Updating object")
 		err = r.Update(ctx, inst)
+		// RoleBindings can't have their Roles changed after they're created 
+		// (see  https://kubernetes.io/docs/reference/access-authn-authz/rbac/#clusterrolebinding-example).
```

Comment:
Change this link to a reference to the issue, and make sure the issue includes this link? The issue will have more context on this problem.

Reply:
done